lib/associations/belongs-to-many.js

```diff
@@ -287,6 +287,16 @@ class BelongsToMany extends Association {
        */
       removeMultiple: 'remove' + plural,
       /**
+       * Un-associate all instances.
+       *
+       * @param {Object} [options] Options passed to `through.destroy`
```

Comment:
Yeah, but all other methods accept an array of instances and only set/remove those exact instances:
```js
setAssociations(instances)
removeAssociations(instances)
```
why does `destroyAssociations` then simply destroy _all_ instances, and not only the ones passed? That confuses me